web-console/src/utils/load-rule.ts

```diff
@@ -41,7 +42,13 @@ export class RuleUtil {
     return (
       rule.type +
       (rule.period ? `(${rule.period})` : '') +
-      (rule.interval ? `(${rule.interval})` : '')
+      (rule.interval ? `(${rule.interval})` : '') +
+      (rule.includeFuture !== false &&
+      (rule.type === 'loadByPeriod' ||
+        rule.type === 'dropByPeriod' ||
+        rule.type === 'broadcastByPeriod')
```

Comment:
this logic should be extracted into a utility function

Reply:
I've made the changes